src/cli/cmds/init.js

```diff
@@ -89,6 +89,15 @@ async function confirmTwaConfig(twaManifest) {
         default: twaManifest.maskableIconUrl || undefined,
         conform: validUrl.isWebUri,
       },
+      shortcuts: {
+        name: 'shortcuts',
+        validator: /y[es]*|n[o]?/,
+        message: 'Include app shortcuts?',
+        description: 'App shortcuts to display for users to users quickly start common or '+
```

Comment:
I think you've got one too many 'users' here.

Reply:
One might argue I have the exact amount of users needed